apps/io-services-cms-webapp/RegenerateServiceKeys/function.json

```diff
@@ -0,0 +1,21 @@
+{
+    "bindings": [
+        {
+            "authLevel": "anonymous",
```

Comment:
are these function being called passing through the API gateway? I'm wondering if is there any reason on keeping them accessible without an host key (anonymous access)

Reply:
The reason is that we have not addressed this issue yet _(we will address it in the next few days)_ and we will do it for all functions.